src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -93,52 +87,48 @@ public virtual int StartTestRun(TestRunCriteria testRunCriteria, ITestRunEventsH
         {
             try
             {
-                if (!this.testHostManager.Shared)
-                {
-                    // Non shared test host requires test source information to launch. Provide the sources
-                    // information and create the channel.
-                    EqtTrace.Verbose("ProxyExecutionManager: Test host is non shared. Lazy initialize.");
-                    var testSources = testRunCriteria.Sources;
-
-                    // If the test execution is with a test filter, group them by sources
-                    if (testRunCriteria.HasSpecificTests)
-                    {
-                        testSources = testRunCriteria.Tests.GroupBy(tc => tc.Source).Select(g => g.Key);
-                    }
+                EqtTrace.Verbose("ProxyExecutionManager: Test host is always Lazy initialize.");
```

Comment:
EqtTrace.Verbose checks internally if verbose is enabled, do we still need to check before hand?